sw/device/lib/dif/dif_rstmgr.c

```diff
@@ -155,6 +159,67 @@ dif_rstmgr_result_t dif_rstmgr_reset_info_clear(const dif_rstmgr_t *handle) {
   return kDifRstmgrOk;
 }
 
+dif_rstmgr_result_t dif_rstmgr_alert_info_set_enabled(
+    const dif_rstmgr_t *handle, dif_rstmgr_toggle_t state) {
+  if (handle == NULL) {
+    return kDifRstmgrBadArg;
+  }
+
+  mmio_region_t base_addr = handle->params.base_addr;
+
+  uint32_t enabled = (state == kDifRstmgrToggleEnabled) ? 0x1 : 0x0;
```

Comment:
I wouldn't bother with the ternary. `? 1 : 0` is the same (from a readability perspective) as `? true : false`.